Dockerfile

```diff
@@ -1,4 +1,17 @@
-FROM horstbaerbel/ccpp-cmake-build-and-test:1.0
+FROM ubuntu:rolling
+
+WORKDIR /
+
+ENV TZ=Europe/Berlin
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
Could you leave a comment on why is this needed please?

Reply:
I've seen this typically first thing when creating OS based images. Or locale even.

But, I'll add a clarification.